modules/tinymce/src/core/main/ts/api/EditorObservable.ts

```diff
@@ -54,12 +54,13 @@ const getEventTarget = (editor: Editor, eventName: string): Node => {
   return editor.getBody();
 };
 
-const isListening = (editor: Editor) => !editor.hidden && !isReadOnly(editor);
+const isListening = (editor: Editor, e: Event) => !editor.hidden && (!isReadOnly(editor) || isReadOnlyAllowedEvent(e));
 
 const fireEvent = (editor: Editor, eventName: string, e: Event) => {
-  if (isListening(editor)) {
+  if (isListening(editor, e)) {
     editor.dispatch(eventName, e);
-  } else if (isReadOnly(editor)) {
+  }
+  if (isReadOnly(editor)) {
```

Comment:
This shouldn't have been changed, as it this meant to be one or the other and never both.

Reply:
reverted sorry, I had interpreted `processReadonlyEvents` as something that changes the read-only events more than something that manages them (and now I figured out that in that case, I should have inverted the 2 ifs :sweat_smile: )